### PR TITLE
Docs: update integrations and verified sources pages

### PR DESCRIPTION
**Pull request by burnash**
This PR:
- Expands the integrations page with brief details on dlt sources and destinations
- Links the REST API source from the sources page

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/index.md [hunk 1]
```diff
@@ -6,14 +6,18 @@ keywords: ['verified source']
 import DocCardList from '@theme/DocCardList';
 import Link from '../../_book-onboarding-call.md';
 
-Pick one of our verified sources that we wrote or maintain ourselves. All of them are constantly tested on real data and distributed as simple Python code so they can be easily customized or hacked.
+Choose from our collection of verified sources, which dlt team developed and maintains. Each source is rigorously tested with real data and provided as Python code for easy customization.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/index.md:9]: >dlt team

and dlt community :D

Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/index.md:9]: > real

should we get rid of it? or our tests data is a real data?

File: docs/website/docs/dlt-ecosystem/verified-sources/index.md [hunk 2]
```diff
@@ -6,14 +6,18 @@ keywords: ['verified source']
...
 
-* Need more info? [Join our Slack community](https://dlthub.com/community) and ask in the tech help channel or <Link/>.
-
-Do you plan to run dlt in production and source is missing? We are happy to build it.
+Planning to use dlt in production and need a source that isn't listed? We're happy to build it for you: <Link />.
 * Source missing? [Request a new verified source](https://github.com/dlt-hub/verified-sources/issues/new?template=source-request.md)
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/index.md:12]:
```suggestion
* Source missing? [Request a new verified source.](https://github.com/dlt-hub/verified-sources/issues/new?template=source-request.md)
```

Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/index.md:12]:
```suggestion
* Source missing? [Request a new verified source.](https://github.com/dlt-hub/verified-sources/issues/new?template=source-request.md)
```

File: docs/website/docs/dlt-ecosystem/verified-sources/index.md [hunk 3]
```diff
@@ -6,14 +6,18 @@ keywords: ['verified source']
...
 * Missing endpoint or a feature? [Request or contribute](https://github.com/dlt-hub/verified-sources/issues/new?template=extend-a-source.md)
+* [Join our Slack community](https://dlthub.com/community) and ask in the tech help channel.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/index.md:14]:
```suggestion
* [Join our Slack community](https://dlthub.com/community) and ask in the technical-help channel.
```

**Review by AstrakhantsevaAA — COMMENTED** [review]
I would also move "30+ databases", "rest api source" and "openapi" on top of the sidebar to make them more visible

File: docs/website/docs/dlt-ecosystem/verified-sources/index.md (hunk 1 above)
Comment (burnash) [reply docs/website/docs/dlt-ecosystem/verified-sources/index.md:9]: > real

I think what was initially intended is that it's real APIs (not real data) vs mock tests. I agree we need to rephrase that.

**Comment by burnash** [comment]
> I would also move "30+ databases", "rest api source" and "openapi" on top of the sidebar to make them more visible

Thanks for the idea, @AstrakhantsevaAA, I tried to addresss it a bit differently in 9feaf25

**Review by AstrakhantsevaAA — APPROVED** [review]
Looks good!

**Pull request merged**